docs/wallet/Desktop-Wallet-Guide.md

```diff
@@ -40,3 +40,68 @@ Due to the nature of UTXO, every time a transaction is made a couple of new Unsp
 Some could consider the desktop wallet slightly less privacy-preserving and secure than the node wallet as it requires you to trust the application’s security and is by default connected to a public node. However, you can build the wallet directly from our GitHub repository and to use a local node or even create your wallet completely offline to mitigate some of these risks.
 
 [latest-release]: https://github.com/alephium/desktop-wallet/releases/latest
+
+## Passphrase
+
+:::info Advanced security feature
+Please, read the following documentation before deciding to use this feature.
+:::
+
+:::tip Important takeaways
+
+- The Passphrase is an advanced security feature that adds an **additional word** that you choose to your existing secret phrase.
+- Using a Passphrase will cause an **entirely new wallet** to be created which cannot be accessed via the secret phrase alone.
+- Apart from adding another layer of security, the Passphrase grants you **plausible deniability** when under duress.
+- If you decide to use a Passphrase, it is vital to store it securely and to **remember it perfectly**, character by character.
+
+:::
+
+Assuming you have created a wallet using the desktop wallet app, you have a list of 24 words referred to as your _secret phrase_. This phrase can be used to restore your wallet and access your funds. If this 24-word secret phrase gets stolen, the attacker will be able to steal your funds. To enhance the security of our desktop wallet users and to prevent loss of funds due to theft of the 24-word secret phrase, we implemented the [BIP39 passphrase][bip39-spec] feature.
+
+The Passphrase is an additional optional 25th word that you are free to choose for yourself. It can be composed of any lower/upper-case character, number, and/or signs and be as long as you want.
+
+Once you create a wallet into the desktop wallet app, the next time you open the app to log in to your wallet, you will be able to provide an additional Passphrase. **It is important to remember that any unique Passphrase will generate and give access to a completely new wallet.**
+
+### Example scenario
+
+#### The "normal wallet"
+
+Let's assume that you created a new wallet, you gave it the name _My Alephium Wallet_ and the password _myVeryStrongPassw0rd_. Let's also assume that the generated 24-word secret phrase is `scan pause slender around cube flavor neck shrug gadget ramp rude lend capable tone nose unhappy gift across cluster minor tragic fever detail script`. You decide to transfer 100 ALPH to it. We'll call this the "normal wallet".
```

Comment:
> oh, so the hidden wallet has the same wallet name displayed as the normal wallet?

yes

> This UX feels a little... "wandering in the dark", but i suppose it's to enhance the "plausible deniablity"? 

The additional word (25th, aka Passphrase) is added on top of the existing 24 words. The 24 words correspond to the wallet named _My Alephium Wallet_. The password of _My Alephium Wallet_ wallet decrypts the 24 words that are stored in the app. The passphrase is the 25th. Combining those together, we get the new wallet. So yes, the wallet name is the same. 

> But it seems like this could potentially create a lot of confusion and nerve wrecking for anyone who inadvertently types the wrong password to access their "normal wallet". (as they would inadvertently be generating a "hidden wallet")

You are absolutely right. We have been debating the implementation of this feature for many months now. [Initial proposals](https://github.com/alephium/desktop-wallet/issues/17#issuecomment-1009992481) suggested to implement it differently to avoid an influx of users complaining that "you stole my funds, the passphrase is correct, but no funds there anymore". The final decision, however, was to stick to the spec of BIP39, write this article, and highly recommend to the users to only use this advanced feature if they know what they are doing.

Reply:
Most of our users will certainly stick to the usual 24 words + password. 
The BIP39 implementation we've done is for those who wants "max" security while waiting for a hardware wallet integration. It's made to be like "wandering in the dark" : if we would guide users more, we would loose plausible deniability.

But when you understand it, it's actually straightforward. 

> But it seems like this could potentially create a lot of confusion and nerve wrecking for anyone who inadvertently types the wrong password to access their "normal wallet".

it's important not to confuse "password" and "optional passphrase". If you inadvertently types the wrong password, you get a "wrong password" error, same as today.